rmw_fastrtps_shared_cpp/src/rmw_node.cpp

```diff
@@ -214,6 +214,40 @@ get_security_file_paths(
   return true;
 }
 
+bool
+get_security_logging_file_path(
+    std::string & security_logging_file_path, const char * node_secure_root)
+{
+  const char * file_names = {"security_logging.xml"};
```

Comment:
Do you actually need the braces, here? Also, given that this is alongside the other security-related files, I'd just call this "logging.xml".

Reply:
Nop braces are definitely optional, I'll remove them and shorten the filename.